CONTRIBUTING.md

```diff
@@ -69,6 +62,16 @@ You can run:
   Python version
 - `tox -e lint` to run lint checks on all code
 
+`tox -e lint` checks the style of the code. It uses [`black`](https://black.readthedocs.io/en/stable/), [`isort`](https://pycqa.github.io/isort/)
+and [`pylint`](http://pylint.pycqa.org/en/latest/) for this purpose. `black` and `isort`-related errors can be fixed automatically by following
+the next steps:
+
+1. `tox -e lint`
+2. `source .tox/lint/bin/activate`
+3. `black .`
+4. `isort .`
+5. `deactivate`
```

Comment:
We should just create a new tox env to do this, so users can simply run `tox -e fix`.